papers/oleksandr_pavlyk/umath/umath.rst

```diff
@@ -13,24 +13,22 @@ functions that are intensively used in financial math applications and machine l
 
 
 One can see that stock Python (pip-installed NumPy from PyPI) on Intel |R| Core |TM| i5 performs
-basic operations such as addition, subtraction, and multiplication just as well as Intel Python,
-but not on Intel |R| Xeon |TM| and Intel |R| Xeon Phi |TM|, where Intel Python adds at least
-another 10x speedup. This can be explained by the fact that basic arithmetic operations in stock
-NumPy are hard-coded AVX intrinsics (and thus already leverage SIMD, but do not scale to other ISA,
-e.g. AVX-512). These operations in stock Python also do not leverage multiple cores (i.e. no
+basic operations such as addition, subtraction, and multiplication just as well as Intel |R| Python,
+but not on Intel |R| Xeon |TM| and Intel |R| Xeon Phi |TM|, where Intel |R| Python provides over 10x speedup. 
```

Comment:
same. Performance claims not discussed in this paper need references to some other supportive evidence